seleniumByTechnocredits/src/seleniumPractise/monali/AlertPractice2.java

```diff
@@ -0,0 +1,49 @@
+package seleniumPractise.monali;
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import util.Init;
+
+public class AlertPractice2 {
+
+	// 3. javaScript Confirmation> Ok/Cancel > Message under the button
+	// Validate the message with OK/Cancel.
+
+	public static void main(String[] args) throws InterruptedException {
+		WebDriver driver = Init.initChromeDriver();
+		driver.get("http://automationbykrishna.com/#");
+		// Basic elements link click
+		driver.findElement(By.linkText("Basic Elements")).click();
+		System.out.println("Basic Elements link is click");
+		Thread.sleep(2000);
+
+		WebElement element = driver.findElement(By.xpath("//button[starts-with(@id,'javascriptConfi')]"));
+		System.out.println("Clicked on Javascript confirmation alert");
+		JavascriptExecutor js = (JavascriptExecutor) driver;
+		js.executeScript("arguments[0].scrollIntoView(true)", element);
```

Comment:
Comment 1 - As scrolling is required in every class, you could have created separate scroll method as utility. If you have written that for practice, that's very good.